tests/test_similarity.py

```diff
@@ -1,15 +1,14 @@
+
```

Comment:
This whole file was reverted to the pre-multiparty version. This is so the PR I make later is strictly additive and we can check that it doesn’t break anything.

The only change I made from the pre-multiparty version is fixing the interaction with Clkhash so it uses the newer API.